### ISSUE TITLE
There is no scroll bar on members page of a workspace

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#25879]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Create a new workspace or open an existing one
2. Click on "Members" link
3. Click on "Invite' button
4. Invite more than 10 users
5. Navigate to members page

## Expected Result: 
There should be a scrollbar on the right side of the pannel
## Actual Result: 
There is no scrollbar

## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.57-2
**Reproducible in staging?:** y
**Reproducible in production?:** n
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/43996225/260d801e-f3f6-4958-91d4-0175990264a6

**Expensify/Expensify Issue URL:**
**Issue reported by:** @Natnael-guchima
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1692789033556139

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by roryabraham** [comment]
Definitely NAB, demoting

**Comment by studentofcoding** [comment]
Is this external or not? @roryabraham

**Comment by Pujan92** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
No scroll bar indicator for workspace members page

### What is the root cause of that problem?
We are not passing the prop `showScrollIndicator` to the `SelectionList` call inside component.

### What changes do you think we should make in order to solve the problem?
Pass `showScrollIndicator` to `SelectionList` call inside workspace members component and other places too if required.
https://github.com/Expensify/App/blob/9a599aa718c48b9c7c68427cafd97810175fbf08/src/pages/workspace/WorkspaceMembersPage.js#L385C26-L385C39

**Comment by jeet-dhandha** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
- No scroll indicator on the right side of the page

### What is the root cause of that problem?
- `showScrollIndicator` is not passed to `SelectionList` component in `WorkspaceMembersPage`.

### What changes do you think we should make in order to solve the problem?
- Passing `showScrollIndicator` prop to `SelectionList` component in `WorkspaceMembersPage` fixes the issue. (This is common in other pages as well, but I'm not sure why we didn't added to this page.)

- Below three modals also uses `SelectionList` component, so we should also pass `showScrollIndicator` prop to them as well. Just to keep the consistency. As all have long list of items.
    - CountrySelectorModal
    - YearPickerModal
    - StateSelectorModal

### What alternative solutions did you explore? (Optional)
- N/A

**Comment by saalimzafar** [comment]
## Proposal

### Problem Statement
The issue at hand involves a missing scroll indicator.

### Root Cause
The problem might be caused by the presence of the `View` Component by which SelectionList Component is wrapped by and the absence of showScrollIndicator prop in SelectionList inside
[App/src/pages/workspace/WorkspaceMembersPage.js]()  file


### Solution
To address this, we can attempt to resolve the issue by removing the `View` Component and by adding showScrollIndicator prop to SelectionList
#### From this:
```jsx
<View style={[styles.w100, styles.mt4, styles.flex1]}>
    <SelectionList
    showScrollIndicator prop is missing
    // other props
    />
</View>
```
#### To this:
```jsx

    <SelectionList
     showScrollIndicator //addition
    // other props
    />
```

### Alternative Solutions Explored
No alternative solutions were explored for this issue.

**Comment by roryabraham** [comment]
@AndrewGable is OOO this week, @sakluger can you take a look at this and see if you think it's worth fixing?

**Comment by Natnael-Guchima** [comment]
@roryabraham the issue is already fixed [here](https://github.com/Expensify/App/pull/25921)

**Comment by Natnael-Guchima** [comment]
@sakluger I disagree with your conclusion. I don't think fixing multiple issues on a single PR equates to that all the bug reports have the same root cause.
